src/base/memory_monitor_sr.hpp

```diff
@@ -40,12 +47,18 @@ namespace cubmem
       memory_monitor &operator = (const memory_monitor &) = delete;
       memory_monitor &operator = (memory_monitor &&) = delete;
 
+    public:
+      size_t get_alloc_size (const char *ptr);
```

Comment:
I think this suggestion is inappropriate because `alloced` can make misunderstanding of word. I think `get_alloc_size` is enough but if changed, I think `get_allocated_size` would be appropriate.

Reply:
We usually use `alloced` to show something already allocated. You can easily find this if you execute `grep -r 'alloced' *` in the cubrid src directory. I think `alloced` is enough to show the meaning and I prefer it. but, if you want to use `allocated`, then that is also good to me.

Reply:
I change it to `allocated`.